<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joystick2Serial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
    <property name="topMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="consoleContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>512</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Consola</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="console">
            <property name="font">
             <font>
              <family>Menlo</family>
              <bold>false</bold>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Menlo'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="placeholderText">
             <string>Todav√≠a no se han recibido datos del puerto serial...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Linea actual:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="currentLine">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearConsole">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Limpiar consola</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="setupContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>324</width>
        <height>512</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <property name="spacing">
        <number>23</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Acerca de...</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="../res/Resources.qrc">:/icon.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Joystick2Serial v1.0.0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Desarrolado por Alex Spataru</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Puerto serial</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="serialDevices">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudRates">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Conectar</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Joystick</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QComboBox" name="joystickList">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="axesContainer" native="true">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="buttonsContainer" native="true">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearConsole</sender>
   <signal>clicked()</signal>
   <receiver>console</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearConsole</sender>
   <signal>clicked()</signal>
   <receiver>currentLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
